src/buffer/out/textBuffer.hpp

```diff
@@ -150,6 +150,11 @@ class TextBuffer final
                                const COLORREF backgroundColor,
                                const std::string& htmlTitle);
 
+    static std::string GenRTF(const TextAndColor& rows,
+                              const int fontHeightPoints,
+                              const std::wstring_view fontFaceName,
+                              const COLORREF backgroundColor);

```

Comment:
`noexcept`

Reply:
`noexcept` is causing the static code analysis to fail. https://github.com/microsoft/terminal/pull/3535/checks?check_run_id=301168873